README.md

```diff
@@ -11,12 +11,22 @@ This plugin will download metadata about Youtube videos using a Youtube API key.
 1. Clone or download this repository.
 1. Ensure you have .NET Core SDK setup and installed.
 1. Build plugin with following command.
-    ```sh
+    ```
     dotnet publish --configuration Release --output bin
     ```
-1. Create folder named `YoutubeMetadata` in the `plugin` directory within `data` directory.
-1. Place the resulting file from step 3 in the `plugins/YoutubeMetadata` folder created in step 4.
-1. If performed correctly you will see the plugin in the dashboard plugin page.
+1. Create folder named `YoutubeMetadata` in the `plugins` directory inside your Jellyfin data
+   directory, usually `/var/lib/jellyfin`. 
```

Comment:
maybe change this to specify on linux? Bonus points if you add in the OSX and Windows paths that are default.

Reply:
Is there not a way to get this plugin into the main directory? I don't have any OSX or Windows install directions. I will generalize this part though.

Reply:
Not sure what you mean by the first sentence. I can make it a todo for myself to find the OSX and Windows ones. Sorry to have pushed that burden onto you.